frontend/src/sass/_color-palettes.scss

```diff
@@ -0,0 +1,107 @@
+// $primary-color: #546E7A;
+
+$mat-httat-palette-primary: (
+  50: #cecbf4,
+  100: #aea9ec,
+  200: #8e86e4,
+  300: #6d63dd,
+  400: #5d52d9,
+  500: #544ac3,
+  600: #4a42ae,
+  700: #413998,
+  800: #2f296d,
+  900: #252157,
+  // A100: #7895a2,
```

Comment:
If the file's not needed, lets remove it.

Reply:
I removed `_color-palettes.scss`